bessctl/commands.py

```diff
@@ -762,9 +767,12 @@ def run_file(cli, conf_file, env_map):
     _run_file(cli, os.path.expanduser(conf_file), env_map)
 
 
-@cmd('add worker WORKER_ID CORE', 'Create a worker')
-def add_worker(cli, wid, core):
-    cli.bess.add_worker(wid, core)
+@cmd('add worker WORKER_ID CORE [SCHEDULER]', 'Create a worker')
+def add_worker(cli, wid, core, scheduler=None):
```

Comment:
Or even better, the Python code should not bother with what the default scheduler is (bessd can choose one if a scheduler is not specified). Currently too many files are coupled with what schedulers are there and what the default is.